### PR TITLE
[ADD] openacademy: Reporting. T#57454

### PR DESCRIPTION
**Pull request by ejdecena**
Exercise 30: Create a report for the Session model.

### CODE REVIEW

File: __manifest__.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
         "views/openacademy_menu.xml",
         "views/openacademy_course_views.xml",
         "views/openacademy_session_views.xml",
+        "report/openacademy_session_report.xml",
         "views/res_partner_views.xml",
```
Comment (ajqn9094) [inline __manifest__.py:18]: I think it is a good practice to maintain an order according to the origin of the file type, this helps to locate files a little faster.

Ej.
- archivos_security
- archivos_data
- archivos_report
- archivos_wizard
- archivos_views

Already in each section you can order by models.

**Comment by ajqn9094** [comment]
The MR name has a extra "]" after the task number

File: __manifest__.py (hunk 1 above)
Comment (ajqn9094) [inline __manifest__.py]: I think it is a good practice to have the files of the same folder in sequence rather than organizing them by models.

```suggestion
        "views/openacademy_menu.xml",
        "views/openacademy_course_views.xml",
        "views/openacademy_session_views.xml",
        "views/res_partner_views.xml",
        "report/openacademy_session_report.xml",
```

Comment (ejdecena) [reply __manifest__.py]: Got it.

**Comment by ejdecena** [comment]
Resolved all of Andy's comments on both PRs :)

cc. @luisg123v @ajqn9094

File: __init__.py [hunk 2]
```diff
@@ -1 +1,2 @@
 from . import models
+from . import report
```
Comment (luisg123v) [inline __init__.py]: - plural
- Directory's `__init__.py` is actually empty

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: __init__.py (hunk 2 above)
Comment (ejdecena) [reply __init__.py]: In the OCA it says singular.

Do I change it to plural?

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#complete-structur

temp
e

Comment (ejdecena) [reply __init__.py]: Done.

File: __manifest__.py [hunk 3]
```diff
@@ -15,6 +15,7 @@
         "views/openacademy_course_views.xml",
         "views/openacademy_session_views.xml",
         "views/res_partner_views.xml",
+        "report/openacademy_session_report.xml",
```
Comment (luisg123v) [inline __manifest__.py:18]: plural, remember we accorded to change it

File: models/openacademy_course.py [hunk 4]
```diff
@@ -10,8 +11,13 @@ class Course(models.Model):
     session_ids = fields.One2many("openacademy.session", string="Sessions", inverse_name="course_id")
 
     _sql_constraints = [
-        ("description_and_title_different", "CHECK(name != description)", "The title and description of the course must be different."),
-        ("unique_course_name", "UNIQUE(name)", "The name of the course must be unique.")
+        ("description_and_title_different",
```
Comment (luisg123v) [inline models/openacademy_course.py:14]: I agree with these changes, but please create a new PR to solve lints, as I commented before. That way, they could be merged immediately and we don't add unrelated changes on this PR/commit

**Review by luisg123v — CHANGES_REQUESTED** [review]
